### PR TITLE
[chore] create all global styles

### PR DESCRIPTION
**Pull request by ccatherinetan**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :palm_tree: What's new in this PR
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
* imported all colors + text styles from figma into `styles/colors` and `styles/text` respectively.

Colors
Screen Shot 2023-11-02 at 2 24 54 PM

Text
 * H1, H2, H3, H4, P
 * by default, their colors are all "dark" 

### :evergreen_tree: Screenshots
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @davidqing6432 at the bottom."

## :deciduous_tree: How to review
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
* checkout my branch `catherine/ijp-65-create-all-global-styles`
* check `styles/colors` and `styles/text`

## :seedling: Next steps 
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
* update everyone's PRs with the new color names + make sure all your styled components inherit from the main H1 etc text styles. 
* do we want to add a "PStrong" text style in the `styles/text` for the bolded text (used frequently on the cases page)? Or would "PStrong" be better suited to be a text style defined in the cases folder? 

### :information_source: Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### :potted_plant: Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @pragyakallanagoudar

### CODE REVIEW

File: src/styles/colors.ts [hunk 1]
```diff
@@ -1,12 +1,21 @@
 const COLORS = {
-  attorneyColor: '#85b4d2',
-  abaBlue: '#0069A9',
+  // Blues
+  primaryBlue: '#0069A9', // tearful-blue
+  secondaryBlue: '#035D85', // grieving
+  tertiaryBlue: '#003453', // drowning
+  lightPrimaryBlue: '#CCE1EE',
+  lighterPrimaryBlue: '#E8F6FF',
```
Comment (jinkang-0) [inline src/styles/colors.ts]: Can we call them `primaryLight` and `secondaryLight` or `...LightBlue` instead?

File: src/styles/colors.ts [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+  // Greys
+  lightGrey: '#F2F2F2',
+  midGey: '#CCC',
+  midDarkGrey: '#818181',
+  darkGrey: '#565656',
   dark: '#292929',
```
Comment (jinkang-0) [inline src/styles/colors.ts]: To stay consistent, can we call these `primaryGrey`, `secondaryGrey` etc?
I think we should either do this or call other colors `lightBlue`, `midBlue` etc

**Review by jinkang-0 — APPROVED** [review]
lgtm

**Review by rahihazra — APPROVED** [review]
SLAY this looks SO GOOD
